Description: do not remove debian directory during build, we need it :)
Forwarded: not-needed
Author: Mikhail Gusarov <dottedmag@dottedmag.net>

--- a/scripts/package/Makefile
+++ b/scripts/package/Makefile
@@ -99,7 +99,7 @@
 	$(MAKE) KBUILD_SRC=
 	$(call cmd,builddeb)
 
-clean-dirs += $(objtree)/debian/
+# clean-dirs += $(objtree)/debian/
 
 
 # tarball targets
